libs/website/builder/src/lib/website-builder.context.tsx

```diff
@@ -141,6 +147,8 @@ export const WebsiteBuilderProvider = memo<
   const parentComponents = useWebsiteBuilder()
   const newComponents = mergeDeepRight(parentComponents, components) as WebsiteBuilderComponents
 
+  console.log(newComponents)
```

Comment:
You want this here?

Reply:
https://github.com/wepublish/wepublish/pull/942/files#diff-60aa3fdb259a650b1fab2bf6dcb3ff07ae1a923b6e3da03beb040c72cf354a6a

i'll be removing this here, is it fine if I merge them together?